tests/test_libutil.py

```diff
@@ -165,3 +166,16 @@ def test_extract_keys_recursive(inp, expected):
     """Test extract keys recursive."""
     result = set(extract_keys_recursive(inp))
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "num,expected",
+    [
+        (0, 1),
+        (1, 1),
+        (22, 2),
+        (335, 3),
```

Comment:
Best to test here are around the border of an extra number so `99` and `100`. As these are the most likely to cause problems.